cypress/e2e/3-testing/access-profile.cy.js

```diff
@@ -0,0 +1,18 @@
+const validEmail = "testemailcypress@stud.noroff.no";
+const validPassword = "testpassword";
+const expectedProfileUrl = "http://127.0.0.1:8080/?view=profile&name=cytest";
+
+describe("valid login", () => {
+  it("allows user to view profile", () => {
+    cy.visit("http://127.0.0.1:8080/");
```

Comment:
```suggestion
    cy.visit("/");
```
That will use cypress server